src/snowflake/cli/plugins/object/stage/commands.py

```diff
@@ -129,6 +130,36 @@ def stage_diff(
     return ObjectResult(str(diff))
 
 
+@app.command("execute", requires_connection=True)
+def execute(
+    stage_path: str = typer.Argument(
+        ...,
+        help="Stage path with files to be execute. For example `@stage/dev/*`.",
```

Comment:
```suggestion
        help="Stage path with files to be execute. For example `@stage/dev/*`.",
        show+default=False,
```
to get rid of "[default: None]" from
```
╭─ Arguments ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ *    stage_path      TEXT  Stage path with files to be execute. For example `@stage/dev/*`. [default: None] [required]                  │
╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

Reply:
Added